src/amo/components/Header/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+
+import Link from 'amo/components/Link';
+import SearchForm from 'amo/components/SearchForm';
+import AuthenticateButton from 'core/components/AuthenticateButton';
+import translate from 'core/i18n/translate';
+import Button from 'ui/components/Button';
+import Icon from 'ui/components/Icon';
+
+import 'mozilla-tabzilla/css/_tabzilla.scss';
+import './styles.scss';
+
+
+export class HeaderBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object.isRequired,
+    isHomePage: PropTypes.bool.isRequired,
+    location: PropTypes.object.isRequired,
+    SearchFormComponent: PropTypes.node.isRequired,
+    query: PropTypes.string,
+  }
+
+  static defaultPropTypes = {
+    SearchFormComponent: SearchForm,
+    isHomePage: false,
+  }
+
+  render() {
+    const {
+      SearchFormComponent,
+      i18n,
+      isHomePage,
+      location,
+      query,
+    } = this.props;
+    const headerLink = (
+      <Link className="Header-title" to="/">
+        <Icon className="Header-addons-icon" name="fox-light" />
+        {
+          // l10n: "Firefox" should not be translated. :-)
```

Comment:
This won't match the translator comment format currently. https://github.com/mozilla/babel-gettext-extractor/blob/master/index.js#L27